src/styles/blocks/_cards.scss

```diff
@@ -0,0 +1,22 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+  width: getContainerWidth($card-width, $card-gap, $card-horizontal-padding);
+  padding: 50px $card-horizontal-padding;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.